src/ui.sortable.multiselection.js

```diff
@@ -178,7 +187,7 @@ angular.module('ui.sortable.multiselection', [])
 
           // Create the helper to act as a bucket for the cloned elements
           var helperTag = item[0].tagName;
-          var helper = angular.element('<' + helperTag + '/>');
+          var helper = angular.element('<' + helperTag + selectionCountAttributes + ' />');
```

Comment:
If you have some extra time please introduce the following change:
```js
var helper = angular.element('<' + helperTag + '/>');
if (uiSelectionCount && selectionCount > 1) {
  helper.addClass('ui-selection-count').attr('data-ui-selection-count',  selectionCount);
}
```
I prefer it from string concatenation :smile: and it shouldn't affect performance since the element is not yet attached to the DOM. As of the naming, I try to use the `ui-` prefix, like jquery-ui plugins do.